Project/Assets/ML-Agents/Examples/SharedAssets/Scripts/DirectionIndicator.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Unity.MLAgentsExamples
+{
+    public class DirectionIndicator : MonoBehaviour
+    {
+        public Transform transformToFollow; //ex: hips or body
+        public Transform targetToLookAt; //target in the scene the indicator will point to
+        public float heightOffset;
```

Comment:
public fields need a capital case I believe

Reply:
no change. current format is what we're using in other scripts so i'll leave as-is.